server/graphql/v1/mutations/collectives.js

```diff
@@ -661,6 +661,15 @@ export async function unarchiveCollective(_, args, req) {
     });
   }
 
+  if (collective.type === types.EVENT) {
+    return collective.update({
+      deactivatedAt: null,
+      isActive: true,
+      HostCollectiveId: collective.ParentCollectiveId,
```

Comment:
That's not correct. HostCollectiveId should be the `HostCollectiveId` of the parent collective.